src/sentry/rules/processing/delayed_processing.py

```diff
@@ -324,7 +324,6 @@ def apply_delayed(project_id: int, *args: Any, **kwargs: Any) -> None:
     """
     # STEP 1: Fetch the rulegroup_to_event_data mapping for the project from redis
     project = Project.objects.get_from_cache(id=project_id)
-    buffer = RedisBuffer()
```

Comment:
In `process_delayed_alert_conditions` should I still be taking `buffer` in or just using the imported buffer?

Reply:
I believe we should be importing buffer here like you've updated rather than initializing a new buffer. it seems like the issue is that calling the constructor is creating a new redis connection which may be to a different instance.